purchase_security/views/purchase_order_views.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="purchase_order_form_own_orders" model="ir.ui.view">
+        <field name="name">purchase.order.form (own orders)</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="groups_id" eval="[(4, ref('group_purchase_own_orders'))]" />
+        <field name="arch" type="xml">
+            <field name="user_id" position="attributes">
+                <attribute name="readonly">1</attribute>
```

Comment:
```suggestion
                <attribute name="readonly">1</attribute>
                <attribute name="force_save">1</attribute>
```

Reply:
Tested on runbot, without `force_save` still works, but I think it's a good point adding it anyway, so I've also included it.